packages/expo-speech/android/src/main/java/expo/modules/speech/LanguageUtils.kt

```diff
@@ -0,0 +1,32 @@
+package expo.modules.speech
+
+import java.util.*
+
+// Lazy load the ISO codes into a Map then transform the codes to match other localization patterns in Expo
+object LanguageUtils {
+  private val countryISOCodes: Map<String, Locale> by lazy {
+    Locale.getISOCountries().map { country ->
+      val locale = Locale("", country)
+      locale.isO3Country.toUpperCase(locale) to locale
+    }.toMap()
+  }
+  private val languageISOCodes: Map<String, Locale> by lazy {
+    Locale.getISOLanguages().map { language ->
+      val locale = Locale(language)
+      locale.isO3Language to locale
+    }.toMap()
+  }
+
+  private fun transformCountryISO(code: String) = countryISOCodes[code]!!.country
+  private fun transformLanguageISO(code: String) = languageISOCodes[code]!!.language
```

Comment:
```suggestion
  private fun transformCountryISO(code: String) = countryISOCodes[code]?.country
  private fun transformLanguageISO(code: String) = languageISOCodes[code]?.language
```
I'm not sure about this `!!` operator here 🤔 
As we have no option here to ensure the `code` String is actually supported, let's try adding nullability and cover it accordingly in the other parts of the code.

Reply:
I was looking for a workaround, but I think it's not worth it.
These methods are unsafe, but they're used only internally and called only with values returned by `Locale` methods. The `country`/`languageISOCodes` are generated from `Locale` values too.
Their purpose is only to "cache" `Locale` ISO cods once (in convenient format) to avoid loading them all with each voice when `SpeechModule.getVoices()` is called.

I think that leaving a comment about the methods being unsafe and how they should be called is sufficient. Or inlining them into callsites, but that would decrease readability (and that won't avoid the need of `!!`)